### DISCUSSION TITLE
Controlling loads in the platform

### DISCUSSION THREAD

**Discussion by shpoudel** [GRIDAPPSD/gridappsd-forum#107] [Q&A]
Hello,

Recently, I was using **EnergyConsumer.p** attribute to control the load power. It seems like the message is sent to the platform, but it's not doing anything. I know the reason for the control signal not working there and it's because of different GLD load definitions. My question is-- Are we just controlling houses in the future and not controlling the loads? I remember there was a transactive app and I am just curious how the control for loads was done in that app?

@poorva1209 @craig8

**Comment by MidrarAdham** [comment]
Hi Shiva,

I'm encountering the same issue. Is there any update?

Thank you.

**Comment by craigpnnl** [comment]
@afisher1 Thoughts?

**Reply by afisher1** [reply]
Hi all,

There is no such SCADA control over a customer's p and q, nor is it physically possible to directly control p and q for a residence directly.  As such energyconsumer.p and energyconsumer.q are not implemented as attributes that applications have control over. Simulated EnergyConsumer load behavior is determined through either normalized load curves or thermodynamic load models modeled by the simulator. I find it hard to justify an application having control over a residential load.

**Reply by shpoudel** [reply]
I am trying to remember why I looked for that feature in April 2021. Andy is correct that such control is not directly possible; however, apps might be exploring load flexibility to provide grid services such as peak demand reduction. Since we don't have a mixture of all possible loads within an EnergyConsumer (for example, we don't have support for EWH), controlling P and Q might offer some insights into how customers can support grid with their flexibility. But, I don't know if this is an important capability to add. 

PSU is working on grid services and they might be interested in such capability.
